homeassistant/components/geo_location/geo_json_events.py

```diff
@@ -0,0 +1,198 @@
+"""
+Generic GeoJSON events platform.
+
+Retrieves current events (typically incidents or alerts) in GeoJSON format, and
+displays information on events filtered by distance to the HA instance's
+location.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/geo_location/geo_json_events/
+"""
+import asyncio
+import logging
+from datetime import timedelta
+from typing import Optional
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.geo_location import GeoLocationEvent
+from homeassistant.const import CONF_RADIUS, CONF_URL, CONF_SCAN_INTERVAL, \
+    EVENT_HOMEASSISTANT_START
+from homeassistant.components.geo_location import PLATFORM_SCHEMA
+from homeassistant.helpers.event import track_time_interval
+
+REQUIREMENTS = ['geojson_client==0.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_EXTERNAL_ID = 'external_id'
+
+DEFAULT_RADIUS_IN_KM = 20.0
+DEFAULT_UNIT_OF_MEASUREMENT = "km"
+
+SCAN_INTERVAL = timedelta(minutes=5)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_URL): cv.string,
+    vol.Optional(CONF_RADIUS, default=DEFAULT_RADIUS_IN_KM):
+        vol.Coerce(float),
+})
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the GeoJSON Events platform."""
+    url = config.get(CONF_URL)
```

Comment:
Use `dict[key]` for required config keys.

Reply:
OK